install_bun.sh

```
@@ -0,0 +1,320 @@
+
+error() {
+    echo -e "${Red}error${Color_Off}:" "$@" >&2
+    exit 1
+}
+
+info() {
+    echo -e "${Dim}$@ ${Color_Off}"
+}
+
+info_bold() {
+    echo -e "${Bold_White}$@ ${Color_Off}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)